src/components/table/table.story.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Table, TableHead, TableBody, TableRow, TableCell } from './index';
+
+storiesOf('Table', module)
+  .add('Default', () => (
+    <Table>
+      <TableHead>
+        <TableCell>id</TableCell>
+        <TableCell>name</TableCell>
+        <TableCell>instrument</TableCell>
+      </TableHead>
```

Comment:
Chose to not require a `TableRow` wrapper around the table cells here in the header (it adds it internally). The downside of that is that with this current API we can't really use ColGroups.

I don't think we've done that a lot (if at all) in Mavenlink, and I judged this API to be okay for now.